templates/js/ai-bot/src/app.js

```diff
@@ -0,0 +1,49 @@
+const { MemoryStorage } = require("botbuilder");
+const path = require("path");
+const config = require("./config");
+
+const {
+  Application,
+  DefaultPromptManager,
+  OpenAIPlanner,
+  AzureOpenAIPlanner,
+} = require("@microsoft/teams-ai");
+
+// Create AI components
+const planner = new OpenAIPlanner({
+  apiKey: config.openAIKey,
+  defaultModel: "gpt-3.5-turbo",
+  useSystemMessage: true,
+  logRequests: true,
+});
+// Uncomment the following lines to use Azure OpenAI
+/**
+const planner = new AzureOpenAIPlanner({
+  apiKey: config.azureOpenAIKey,
+  endpoint: config.azureOpenAIEndpoint,
+  defaultModel: "gpt-35-turbo",
```

Comment:
gpt-3.5-turbo or gpt-35-turbo?

Reply:
OpenAI uses `gpt-3.5-turbo` and Azure OpenAI uses `gpt-35-turbo` (https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#gpt-35)